data/data-pipeline/data_pipeline/etl/sources/tree_equity_score/etl.py

```diff
@@ -83,8 +83,52 @@ def transform(self) -> None:
             pd.concat(tes_state_dfs), crs=tes_state_dfs[0].crs
         )
 
+        # rename ID to Tract ID
+        self.df.rename(
```

Comment:
this might raise a warning about changing the dataframe in place (instead of copying it), but if you're not getting an error or warning and the rename is successful, then all good!

Reply:
Indeed, I have not been receiving any errors or warnings from Pandas 👍